### ISSUE TITLE
Option to hide OSD when pausing

### ISSUE THREAD

**Issue by ak-988** [jellyfin/jellyfin-androidtv#3924]
### Describe the feature you'd like

Many thanks to everyone for their hard work on Jellyfin projects. One feature I miss from Kodi is the option to hide the OSD when playback is paused. 

Sometimes, we'll be immersed in a movie and need to pause it briefly. Seeing there's actually only fifteen minutes left kind of changes my viewing experience afterwards almost like a spoiler. 

Ideally, and to match Kodi's feature, it would be possible to have the OSD hidden when pausing from the explicitly marked "pause" button on my Sony remote and allow it to be shown when pausing via the circular "select" button that also currently pauses Jellyfin. I realize this is a very specific little thing but it's really the only unpleasant part of my experience with this wonderful app.

**Comment by SteveDinn** [comment]
I would like to see this as well. Perhaps an option for pausing on its own to show no overlay, but when paused, pressing up on the directional pad would show things, like it does when playing.

**Comment by DrMcCoy** [comment]
This would not only be useful for improved immersion, it would also help being able to actually inspect the frame. 

Sometimes, something is only shown for a short time, too quick to get a detailed view. I like to pause then, but the controls make it difficult. 

Additionally, I'm kinda learning Japanese, a
I like to pause on any written things during an anime episode or the like, to try to read what's written there. And, heck sometimes these two things come together: sometimes something written is only shown for a short time and even reading the subs for the written things, say a phone messenger conversation, is not possible with how quickly it's gone from the screen.

**Comment by mschumacher69** [comment]
Press the back button on your remote while the video is paused and the osd would go away.

But what irritates me the most about this osd is that it doesn't disappear right away after you press play, it should disappear right away when you press play in order not to miss any details in the scene.

**Comment by DrMcCoy** [comment]
> Press the back button on your remote while the video is paused and the osd would go away.

Oh.

Yeah, that works. I guess I never even tried that. Well, here's me with egg on my face.

**Comment by SteveDinn** [comment]
> Press the back button on your remote while the video is paused and the osd would go away.

I always find this super awkward because I inevitably wait just long enough so that when I go to push the back button, OSD has *just* disappeared and I end up exciting out to the menu. This happens more often than I care to admit. 

I would rather that it took an 'up'  press to get it to appear. Before that it should only play/pause with the centre button.

**Comment by mschumacher69** [comment]
Yeah, I usually press back right after I hit play, but I mean you don't need the OSD after hitting play anymore, it should disappear right away after hitting play.

**Comment by tim-tx** [comment]
I sometimes watch replays of sports events and sometimes I seek forward to skip some dead part. The back button trick doesn't help if you're trying to seek. I've been holding my arm up to block the bottom third of the screen. Option to disable OSD bar would be great.

**Comment by JustR0b0t** [comment]
I got impatient and sort of implemented this feature in my own fork of the app.

The code works, so if anybody is interested in implementing it as an option in the settings, feel free to do so.

This code just ignores the OSD if certain buttons like Pause or Rewind are pressed. It might lead to bugs elsewhere, but it works with my Nvidia Shield TV Pro remote on a version I compiled myself. (based on v0.19.3)

Just change the code in **CustomPlaybackOverlayFragment.java** with the following changes:

```
   private View.OnKeyListener keyListener = new View.OnKeyListener() {
        @Override
        public boolean onKey(View v, int keyCode, KeyEvent event) {
            if (event.getAction() == KeyEvent.ACTION_DOWN) {

                // Only show overlay if certain buttons are not pressed
                if (keyCode != KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE
                        && keyCode != KeyEvent.KEYCODE_MEDIA_REWIND
                        && keyCode != KeyEvent.KEYCODE_MEDIA_FAST_FORWARD) {
                    if (!mGuideVisible) {
                        leanbackOverlayFragment.setShouldShowOverlay(true); 
                    }else {
                    leanbackOverlayFragment.setShouldShowOverlay(false);
                    leanbackOverlayFragment.hideOverlay();
                    }
                }else{
                    leanbackOverlayFragment.setShouldShowOverlay(false);
                    leanbackOverlayFragment.hideOverlay();
                }
```

**Comment by voirfilm-tv** [comment]
Hello @JustR0b0t, where is this famous file located?

**Comment by JustR0b0t** [comment]
@voirfilm-tv Its the file in app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

**Comment by theniwo** [comment]
Did you made a pr on the main git for this? It's a requested feature and I think no one likes to manually update the app everytime just for this little code change. 
Alas this is hardcoded, an option to switch this behaviour would be preferrable.
